src/main/resources/templates/score/team.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="cz" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+    <title>Tým - Detailní score</title>
+</head>
+<body>
+
+<div class="container">
+    <h1>Tým <span th:text="${team.name}"></span> - Stav</h1>
+    <h2>Šifry:</h2>
+    <div class="table-responsive">
+        <table class="table" border="1">
+            <thead>
+            <td align="center">Id</td>
```

Comment:
Za me bych odstranil ten atribut align a pripadne to pak vyresil pomoci CSS.

Reply:
mám to vyhodit všude ve všech výskytech, a i v tom html, co jsem dělala předtím?

Reply:
Ve výskytech prosím. Ale stačí jen v těch souborech, co jsou v tomto requestu.

Reply:
opraveno